Tools/python/job_management.py

```diff
@@ -105,6 +106,15 @@ def convert_catalog(file_list,as_dict=True):
 # HTCondor interface for job submission and tracking
 #############################################################
 
+job_status = {
+           1:'idle',
+           2:'running',
+           3:'removed',
+           4:'completed',
+           5:'held',
+           6:'transferring output',
+           7:'suspended',
+        }
 
 base_job_properties = {
```

Comment:
This code is duplicated below. The defaults should be set by a call to `setup_schedd('T3')` when the module is loaded.